deploy/charts/rook-ceph-cluster/templates/cephcluster.yaml

```diff
@@ -15,5 +15,4 @@ spec:
     externalMgrPrometheusPort: {{ toYaml .Values.monitoring.externalMgrPrometheusPort }}
 {{- end }}
 {{- end }}
-
-{{ toYaml .Values.cephClusterSpec | indent 2 }}
+{{ include "rook-ceph-cluster.cephClusterSpec" . | indent 2 }}
```

Comment:
How about adding a comment with this line to point out that it was defined in the helper templates?

Reply:
Done (I've amended my commit to keep it simple)